### PR TITLE
RHBA-611 - The system property org.jbpm.rm.init.timer should be set, JBPM-7071 - Get workitems from REST API call doesn't show up all

### PR DESCRIPTION
**Pull request by mswiderski**

### CODE REVIEW

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+public class EventSubProcessServiceImplTest extends AbstractKieServicesBaseTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(EventSubProcessServiceImplTest.class);
+
+    private List<DeploymentUnit> units = new ArrayList<DeploymentUnit>();
+
+    @Before
+    public void prepare() {
+    	configureServices();
+    	logger.debug("Preparing kjar");
```
Comment (MarianMacik) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java]: Formattting.

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        File pom = new File("target/kmodule", "pom.xml");
+        pom.getParentFile().mkdir();
+        try {
+            FileOutputStream fs = new FileOutputStream(pom);
+            fs.write(getPom(releaseId).getBytes());
+            fs.close();
+        } catch (Exception e) {
+
+        }
+		KieMavenRepository repository = getKieMavenRepository();
```
Comment (MarianMacik) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java]: Formatting.

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java (hunk 1 above)
Comment (MarianMacik) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java]: Formatting.

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java (hunk 2 above)
Comment (MarianMacik) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java]: Formatting.

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        assertEquals("Human Task", wis.get(0).getName());
+        assertEquals("taxiUser", wis.get(0).getParameter("NodeName"));
+
+        assertEquals("Human Task", wis.get(1).getName());
+        assertEquals("towingUser", wis.get(1).getParameter("NodeName"));
+        
+    	processService.abortProcessInstance(processInstanceId);
+
+    	pi = processService.getProcessInstance(processInstanceId);
+    	assertNull(pi);
```
Comment (MarianMacik) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/EventSubProcessServiceImplTest.java]: Again formatting weird a little bit.

**Comment by MarianMacik** [comment]
BTW what about making a new test class called ETimerServiceTest and putting the new test there? In future we can add other timer related tests there (though now I am not sure what use cases :) maybe some checks directly on EJBTimerService?). Just thinking...

**Comment by mswiderski** [comment]
@MarianMacik comments applied.

regarding separate test, I think we better add new test class when is really needed, for this simple test case I don't think it's worth it as it will then setup all over again which takes a bit.

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins execute full downstream build

**Comment by MarianMacik** [comment]
@mswiderski Good point, actually.

**Pull request merged**